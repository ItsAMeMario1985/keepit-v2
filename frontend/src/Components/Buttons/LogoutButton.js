import PropTypes from 'prop-types'
import styled from 'styled-components/macro'
import { LogoutIcon } from '../Icons'
import { useHistory, useLocation } from 'react-router-dom'

export default function LogoutButton({ onClick }) {
  const history = useHistory()

  return <LogoutIcon onClick={onClick}></LogoutIcon>
}
