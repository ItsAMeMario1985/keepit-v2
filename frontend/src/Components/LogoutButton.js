import PropTypes from 'prop-types'
import styled from 'styled-components/macro'
import { ReactComponent as LogoutIcon } from '../Assets/logout.svg'
import { useHistory, useLocation } from 'react-router-dom'

export default function LogoutButton({ onClick }) {
  const history = useHistory()

  return <LogoutIcon onClick={onClick}></LogoutIcon>
}
