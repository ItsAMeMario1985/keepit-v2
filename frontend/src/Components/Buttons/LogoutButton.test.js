import { render } from '@testing-library/react'
import LogoutButton from './LogoutButton'
describe('LogoutButton', () => {
  it('renders correctly', () => {
    const { container } = render(<LogoutButton />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
