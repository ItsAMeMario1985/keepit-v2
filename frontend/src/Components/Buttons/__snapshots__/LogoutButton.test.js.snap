// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LogoutButton renders correctly 1`] = `
<svg>
  logout.svg
</svg>
`;
